projects/ngx-skeleton-loader/src/lib/ngx-skeleton-loader.component.spec.ts

```diff
@@ -98,37 +98,13 @@ describe('NgxSkeletonLoaderComponent', () => {
   describe('When the component uses default configuration', () => {
     beforeEach(waitForAsync(() => {
       fixture = TestBed.configureTestingModule({
-        declarations: [ContainerComponent, NgxSkeletonLoaderComponent],
+        declarations: [ContainerComponent],
+        imports: [NgxSkeletonLoaderComponent],
         providers: [{ provide: PLATFORM_ID, useValue: 'browser' }],
       }).createComponent(ContainerComponent);
       fixture.detectChanges();
     }));
 
-    it('should console 3 errors if `animation`, `appearance` and `count` receives invalid options in development mode', () => {
```

Comment:
These tests are still relevant for consumers. Could you add them back, and the code to support those tests as well, please?